src/database/schema/ChatSchema.ts

```diff
@@ -72,8 +76,9 @@ class ChatSchema implements BaseDbSchema {
         status,
         created_at,
         updated_at,
-        raw_json
-      ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?);`;
+        raw_json,
+        attachment_json
+      ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?);`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query has been updated to include `attachment_json`, but there's no check to ensure that `attachment_json` is a valid JSON string. Consider adding a check before executing the query.
```diff
-        raw_json,
-        attachment_json
+        raw_json = ?,
+        attachment_json = CASE WHEN json_valid(?) THEN ? ELSE NULL END
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->